src/v1/sources/shopify/identifierEventsLayer.js

```diff
@@ -0,0 +1,57 @@
+const { IDENTIFIER_EVENTS, NO_OPERATION_SUCCESS } = require('./config');
+const stats = require('../../../util/stats');
+const { getLineItemsToStore } = require('./commonUtils');
+const { RedisDB } = require('../../../util/redis/redisConnector');
+const logger = require('../../../logger');
+
+const IdentifierEventLayer = {
+  isIdentifierEvent(event) {
+    return IDENTIFIER_EVENTS.includes(event?.event);
```

Comment:
```suggestion
    return IDENTIFIER_EVENTS.includes(event.event);
```
Can we update the parameter name to message from event?